app/src/main/java/com/infomaniak/mail/ui/main/newMessage/NewMessageViewModel.kt

```diff
@@ -74,16 +75,19 @@ class NewMessageViewModel : ViewModel() {
     }
 
     private fun configureDraft(draftUuid: String? = null) = viewModelScope.launch(Dispatchers.IO) {
-        val uuid = RealmDatabase.mailboxContent().writeBlocking {
-            return@writeBlocking if (draftUuid == null) {
+        RealmDatabase.mailboxContent().writeBlocking {
+            val uuid = if (draftUuid == null) {
                 createDraft()
             } else {
                 updateLiveData(draftUuid)
                 draftUuid
             }
-        }
 
-        currentDraftUuid.postValue(uuid)
+            DraftController.setDraftSignature(uuid, this)
+            mailBody = DraftController.getDraft(uuid, this)?.body ?: ""
```

Comment:
The `setDraftSignature` method should be used directly to retrieve the draft that has been updated, but it should be renamed to `setAndGetDraftSignature`.

Reply:
Will be fixed in #314.